### ISSUE TITLE
Load and Explore the Data

### ISSUE THREAD

**Issue by Nishant001-jyo** [Nishant001-jyo/Nishant001-jyo#4]
Create a new file, e.g., iris_classifier.py or a Jupyter notebook.
`from sklearn.datasets import load_iris
import pandas as pd
import seaborn as sns
import matplotlib.pyplot as plt

iris = load_iris()
X = iris.data  # Features
y = iris.target  # Labels

# Convert to DataFrame for exploration
df = pd.DataFrame(X, columns=iris.feature_names)
df['species'] = iris.target_names[y]

# Explore
print(df.head())  # First 5 rows
print(df.describe())  # Summary stats
print(df['species'].value_counts())  # Class balance

# Visualize
sns.pairplot(df, hue='species')
plt.show()`
